lib/omniship/dhlgm/track/package.rb

```diff
@@ -50,30 +50,20 @@ def scheduled_delivery_date
 
         # this is actually an indicator that the DHLGM shipping facility has received the package
         def has_left?
-          self.activity.each {|activity|
-            if activity.code.to_i >= 100
-              return true
-            end
-          }
-          return false
+          activity.any? {|activity| activity.code.to_i >= 100 }
```

Comment:
shouldn't we make sure ```activity``` isn't empty? b/c of the whole empty array .any? == true?